src/SwiftOutputChannel.ts

```diff
@@ -0,0 +1,50 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the VSCode Swift open source project
+//
+// Copyright (c) 2021 the VSCode Swift project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of VSCode Swift project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import * as vscode from 'vscode';
+import { Disposable } from 'vscode-jsonrpc';
+
+export class SwiftOutputChannel implements Disposable {
+    private channel: vscode.OutputChannel;
+
+    constructor() {
+        this.channel = vscode.window.createOutputChannel("Swift");
+    }
+
+    dispose() {
+        this.channel.dispose();
+    }
+
+    log(message: string, folder?: string) {
```

Comment:
Perhaps `folder` can just be of type `WorkspaceFolder`, so we don't always have to find its path when we call `log`?

Reply:
I think `folder` is probably the wrong name. It is a just a name to indicate where a log message is coming from. This might not be a folder. 

`label`?